ibm-mq-mixin/dashboards/queue-manager-overview.libsonnet

```diff
@@ -0,0 +1,1287 @@
+local g = (import 'grafana-builder/grafana.libsonnet');
```

Comment:
I believe this import is not used here or in the other .libsonnet dashboards and can be removed.